#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_multiple_parameters.dpatch by Iain Lane <iain@orangesquash.org.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This patch fixes a compiler error due to multiple parameters having the same name

@DPATCH@
diff -urNad mpeg4ip-1.6dfsg~/server/mp4live/video_util_resize.h mpeg4ip-1.6dfsg/server/mp4live/video_util_resize.h
--- mpeg4ip-1.6dfsg~/server/mp4live/video_util_resize.h	2005-03-25 20:15:31.000000000 +0000
+++ mpeg4ip-1.6dfsg/server/mp4live/video_util_resize.h	2008-08-29 19:21:37.000000000 +0100
@@ -90,7 +90,7 @@
 
 void CopyYuv(const uint8_t *fY, const uint8_t *fU, const uint8_t *fV,
 	     uint32_t fyStride, uint32_t fuStride, uint32_t fvStride,
-	     uint8_t *tY, uint8_t *tU, uint8_t *fV,
-	     uint32_t tyStride, uint32_t tvStride, uint32_t tvStride,
+	     uint8_t *tY, uint8_t *tU, uint8_t *tV,
+	     uint32_t tyStride, uint32_t tuStride, uint32_t tvStride,
 	     uint32_t w, uint32_t h);
 #endif
